src/sentry/south_migrations/0478_add_model_IncidentSeen.py

```diff
@@ -0,0 +1,1344 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
Let's mark this as `is_dangerous = True`. Then you can just ask #ops to run this migration manually.

You'll need to provide them with the DDL to actually run, you can generate it with:
`sentry django south_migrate sentry 0478 --verbosity=2`

Reply:
Cool - posting here for posterity
```
CREATE TABLE "sentry_incidentseen" ("id" bigserial NOT NULL PRIMARY KEY, "incident_id" bigint NOT NULL, "user_id" integer NOT NULL, "last_seen" timestamp with time zone NOT NULL);
ALTER TABLE "sentry_incidentseen" ADD CONSTRAINT "incident_id_refs_id_74273623" FOREIGN KEY ("incident_id") REFERENCES "sentry_incident" ("id") DEFERRABLE INITIALLY DEFERRED;
CREATE INDEX "sentry_incidentseen_incident_id" ON "sentry_incidentseen" ("incident_id");
ALTER TABLE "sentry_incidentseen" ADD CONSTRAINT "user_id_refs_id_d6a00348" FOREIGN KEY ("user_id") REFERENCES "auth_user" ("id") DEFERRABLE INITIALLY DEFERRED;
```

Reply:
Looks like the `UNIQUE INDEX` is missing there?

Reply:
Thank you for spotting - updated

Reply:
Looks good! You can post in ops and ask them to run that sql, then afterwards we can merge/deploy